examples/servicegraph/deploy/kube/servicegraph.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: servicegraph
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: servicegraph
+    spec:
+      containers:
+      - name: servicegraph
+        image: localhost:5000/servicegraph
```

Comment:
this seems not ideal. can we have this be a gcr.io/istio-testing/servicegraph (or similar)? even if we don't push it to gcr.io or docker-hub, we can still run the build and tag it as if we were going to...